src/webui/www/private/views/preferences.html

```diff
@@ -999,6 +999,14 @@
                     <input type="text" id="outgoingPortsMax" style="width: 15em;" />
                 </td>
             </tr>
+            <tr>
+                <td>
+                    <label for="UPnPLeaseDuration">QBT_TR(UPnP lease duration [0: Permanent lease]:)QBT_TR[CONTEXT=OptionsDialog]</label>
```

Comment:
I think you forgot to add the "?" link.

Reply:
Oops! Added it to the GUI but not WebUI. Added.